v1/metrics.yaml

```diff
@@ -232,6 +232,59 @@ paths:
               uri: '{{options.host}}/pageviews/top/{project}/{access}/{year}/{month}/{day}'
       x-monitor: false
 
+  /unique-devices/{project}/{access-site}/{granularity}/{start}/{end}:
+    get:
+      tags:
+        - Unique devices data
+      summary: Get unique devices count per project
+      description: |
+        Given a project and a date range, returns a timeseries of unique devices counts. You need to specify a project, and can filter by accessed site (mobile or desktop). You can choose between daily and hourly granularity as well
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: project
+          in: path
+          description: The name of any Wikimedia project formatted like {language code}.{project name}, for example en.wikipedia. You may pass en.wikipedia.org and the .org will be stripped off. For projects like commons without language codes, use commons.wikimedia
+          type: string
+          required: true
+        - name: access-site
```

Comment:
Sorry, I wrote this quickly and it's more confusing than that.

 * `access` is the way our content was accessed from the client point of view.  So it does include the desktop site (`en.wikipedia.org`), the mobile site (`en.m.wikipedia.org`), but also the mobile app (however that works, we track mw api requests, restbase, etc.)
 * `access-site` doesn't know anything about the client, and so it can only be `en.wikipedia.org` which we call desktop, or `en.m.wikipedia.org` which we call mobile

This is due to how the cookie that powers the `unique_devices` data works, and I agree now that the two parameters have almost the same semantics, which is why we chose a similar name.  But we didn't want the same exact name because of the mobile-app difference and potentially other differences as we expand each metric (like we may have facebook, iOS, or google knowledge graph choices for the `access` parameter, but we'll be restricted to our own domains for the unique devices dataset).